src/frontend/styles/components/sidebar.pcss

```diff
@@ -0,0 +1,213 @@
+.docs-sidebar {
+  --sidebar-width: 100vw;
+
+  width: var(--sidebar-width);
+    
+  @media (--desktop) {
+    --sidebar-width: 344px;
```

Comment:
maybe it would be better to move it to the `vars.pcss` under the `--layout-sidebar-width`?